test/data/migration-config.yml

```diff
@@ -1,3 +1 @@
-migration:
-  target:
-    product: foo
+migration_product: sles
```

Comment:
again I don't think introducing our own names and maintain a mapping table is not a good idea